scripts/lib-franklin.js

```diff
@@ -428,6 +428,24 @@ export function buildBlock(blockName, content) {
   return (blockEl);
 }
 
+/**
+ * Gets the configuration for the given block, and also passes
+ * the config through all custom patching helpers added to the project.
+ *
+ * @param {Element} block The block element
+ * @returns {Object} The block config (blockName, cssPath and jsPath)
+ */
+function getBlockConfig(block) {
+  const { blockName } = block.dataset;
+  const cssPath = `${window.hlx.codeBasePath}/blocks/${blockName}/${blockName}.css`;
+  const jsPath = `${window.hlx.codeBasePath}/blocks/${blockName}/${blockName}.js`;
+  const original = { blockName, cssPath, jsPath };
+  return window.hlx.patchBlockConfig.reduce(
+    (config, fn) => (typeof fn === 'function' ? fn(config, original) : config) || config,
+    { blockName, cssPath, jsPath },
+  );
```

Comment:
```suggestion
  return window.hlx.patchBlockConfig
    .filter((fn) => typeof fn === 'function')
    .reduce(
    (config, fn) => fn(config, original),
    { blockName, cssPath, jsPath },
  );
```

This makes it a bit more readable.